qa/TL0_cpu_only/test.sh

```diff
@@ -31,7 +31,7 @@ test_body() {
         exit 1
     fi
 
-    "$FULLPATH" --gtest_filter="*CpuOnlyTest*"
+    "$FULLPATH" --gtest_filter="*CpuOnly*:*CApi*/0.*-*0.UseCopyKernel:*ForceNoCopyFail:*daliOutputCopySamples"
```

Comment:
Does `0` mean CPU?

Reply:
Yes - unfortunately the name of the TypeParam is not a part of the name, so you can't match against that.